radio/src/Makefile

```diff
@@ -404,7 +404,11 @@ else
 endif
 TRANSLATE = -$(PYTHON) ../util/translate.py
 
-VERSION = 2.2.0$(OPENTX_VERSION_SUFFIX)
+VERSION_MAJOR    = 2
```

Comment:
Yes but I meant for the windows server, where cut etc are not available so easily!